sw/airborne/modules/rl_obstacle_avoidance/log_to_file.c

```diff
@@ -0,0 +1,137 @@
+//
+// Created by geart on 5-11-18.
```

Comment:
This logger system is interesting, so I would suggest to make it independent of the rest and place it in the `modules/loggers` folder.
You should also add the license header.